--- snapcast-0.19.0/client/Makefile	2020-03-14 11:22:04.000000000 +0100
+++ snapcast-0.19.0-makefiles/client/Makefile	2020-05-14 07:40:28.699731588 +0200
@@ -83,9 +83,8 @@
 
 else
 
-CXX       = g++
 CXXFLAGS += -pthread -DHAS_VORBIS -DHAS_ALSA -DHAS_AVAHI -DHAS_DAEMON -DHAS_SOXR
-LDFLAGS  += -lrt -lasound -lvorbis -lavahi-client -lavahi-common -latomic
+LDFLAGS  += -lrt -lasound -lvorbis -lavahi-client -lavahi-common -logg
 OBJ      += ../common/daemon.o player/alsa_player.o browseZeroConf/browse_avahi.o
 
 endif
diff -Naur snapcast-0.19.0/server/Makefile snapcast-0.19.0-makefiles/server/Makefile
--- snapcast-0.19.0/server/Makefile	2020-03-14 11:22:04.000000000 +0100
+++ snapcast-0.19.0-makefiles/server/Makefile	2020-05-14 07:40:57.194712585 +0200
@@ -88,9 +88,8 @@
 
 else
 
-CXX       = g++
 CXXFLAGS += -DHAS_AVAHI -DHAS_DAEMON -pthread
-LDFLAGS  += -lrt -lavahi-client -lavahi-common -latomic
+LDFLAGS  += -lrt -lavahi-client -lavahi-common -logg
 OBJ      += ../common/daemon.o publishZeroConf/publish_avahi.o 
 
 endif
